app/services/podcast/feed.rb

```diff
@@ -0,0 +1,81 @@
+require "rss"
+
+module Podcast
+  class Feed
+    def initialize
+      @config = Config.new
+
+      @feed = RSS::Maker.make("2.0") do |feed|
+        set_channel_from_config(feed.channel)
+        set_itunes_from_config(feed.channel)
+
+        feed.channel.itunes_categories.new_category do |category|
+          category.text = @config.category[0]
+          category.new_category.text = @config.category[1]
+        end
+
+        radios = [
+          {
+            title: "title1",
+            path: "/radios/1",
+            published_at: "Mon, 02 Oct 2017 01:34:09 +0000",
+            description: "ほげほげほげほげげ",
+            duration: 600,
+          },
+          {
+            title: "title2",
+            path: "/radios/2",
+            published_at: "Mon, 02 Oct 2017 01:34:09 +0000",
+            description: "ふがふがふがふがふがが",
+            duration: 6000,
+          },
+        ]
+
+        radios.each do |radio|
+          set_item(feed.items, radio)
+        end
+      end
+    end
+
+    def generate
+      @feed.to_xml
+    end
+
+    private
+
+      def set_channel_from_config(channel)
+        channel.title               = @config.podcast_name
+        channel.link                = @config.application_url
+        channel.description         = @config.podcast_description
+        channel.language            = @config.language
+        channel.sy_updatePeriod     = @config.update_period
+        channel.sy_updateFrequency  = @config.update_frequency
+        channel.managingEditor      = "#{@config.owner_email} (#{@config.owner_name})"
+        channel.lastBuildDate       = Time.now.utc # 一番後のpublished_atの値が入る
+      end
+
+      def set_itunes_from_config(channel)
```

Comment:
__[RuboCop]__

Naming/AccessorMethodName: Do not prefix writer method names with `set_`.


<!-- Comment by [SideCI](https://sideci.com) -->